plugins/search-react/src/components/SearchResultLimiter/SearchResultLimiter.stories.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { ComponentType } from 'react';
+import { Grid } from '@material-ui/core';
+
+import { TestApiProvider } from '@backstage/test-utils';
+
+import { searchApiRef, MockSearchApi } from '../../api';
+import { SearchContextProvider } from '../../context';
+
+import { SearchResultLimiter } from './SearchResultLimiter';
+
+export default {
+  title: 'Plugins/Search/SearchResultLimiter',
+  component: SearchResultLimiter,
```

Comment:
by @emmaindal 
> I really liked that you implemented this component. I wonder if we should make it part of the pagination? I'm thinking something like this https://mui.com/material-ui/react-pagination/#table-pagination What do you think?
https://github.com/backstage/backstage/pull/13950#discussion_r985434212

Reply:
Yes, I've looked at this table pagination component before but wasn't sure we have all the props needed to calculate the number of pages using the page cursor. So I created this simple limiter. Do you know if we can do this with the page cursor? We can confirm together if you have a moment today 🙂

Reply:
Done!